src/components/Button/index.js

```diff
@@ -51,27 +48,38 @@ const StyledButton = styled.button`
         ? space[44]
         : space[56]
       : 'auto'};
-  cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
+  cursor: ${props => (props.disabled ? 'default' : 'pointer')};
   height: ${props => (props.size === 'small' ? space[44] : space[56])};
-  transition: color ${transition}, background-color ${transition},
-    text-shadow ${transition};
+  transition: color 100ms ${easing.easeInOutCubic},
+    background-color 100ms ${easing.easeInOutCubic},
+    text-shadow 100ms ${easing.easeInOutCubic};
 
   svg {
     filter: drop-shadow(${shadow.text});
   }
 
+  ${props =>
+    props.variant === 'primary' &&
+    css`
+      background-color: ${color.earth};
+      color: white;
+      text-shadow: ${shadow.text};
+    `};
+
   ${props =>
     props.variant === 'secondary' &&
     css`
-      background-color: ${props.active ? color.earth : color.stardustLight};
-      color: ${props.active ? 'white' : color.space};
+      background-color: ${props.active ? color.earth : '#F0FBFE'};
```

Comment:
Just a curiosity here and on line 80, we don't want to add these colors to `theme.js`?

Reply:
We decided against that, because we want to restrict the options developers can choose from. These colours are an exception for these specific button-variants.